src/BaselineOfFamix/BaselineOfFamix.class.st

```diff
@@ -34,7 +35,7 @@ BaselineOfFamix >> baseline: spec [
 
 			"Packages"
 			spec
-				package: 'Moose-Query' with: [ spec requires: #('Fame' 'DeepTraverser' 'CollectionExtensions') ];
+				package: 'Moose-Query' with: [ spec requires: #('Fame' 'DeepTraverser' 'CollectionExtensions' 'TreeQuery') ];
```

Comment:
Do you plan to use TreeQuery in Moose-Query?
Or is it because you wanted to create the dependency?

Reply:
I want to create a dependency.

I would like to use it for MooseQuery but I guess the performances will stop us.